gateway-provider-security-jwt/src/main/java/org/apache/knox/gateway/provider/federation/jwt/filter/SSOCookieFederationFilter.java

```diff
@@ -92,6 +100,18 @@ public void init( FilterConfig filterConfig ) throws ServletException {
       publicKey = CertificateUtils.parseRSAPublicKey(verificationPEM);
     }
 
+    /* get unauthenticated paths list */
+    String unAuthPathString = filterConfig.getInitParameter(SSO_UNAUTHENTICATED_PATHS_PARAM);
+    /* if no list specified use default value */
+    if (StringUtils.isBlank(unAuthPathString)) {
+      unAuthPathString = DEFAULT_SSO_UNAUTHENTICATED_PATHS_PARAM;
+    }
+
+    final StringTokenizer st = new StringTokenizer(unAuthPathString, ";");
```

Comment:
Could we support ',' in addition to ';' as a delimiter here? I've personally been burned by using the incorrect delimiter in some provide config property values.

Reply:
+1 to Phil's suggestion

Reply:
Yeah that is a good idea, i was conflicted whether to support "," decided not to include to be in line with other config props. 
I will update it to support ",".